### PR TITLE
Add docker image for Kubernetes dashboard

### PR DESCRIPTION
**Pull request by nanoscopic**

### CODE REVIEW

File: kube-dash-image/kube-dash-image.kiwi.ini [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      image="docker" 
+      derived_from="obsrepositories:/_BASEIMAGE_">
+      <containerconfig 
+        name="_PRODUCT_/kube-dash" 
+        tag="%%TAG%%" 
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/kubernetes-dashboard"/>
+      </containerconfig>
+    </type>
+    <version>1.0.1</version>
```
Comment (davidcassany) [inline kube-dash-image/kube-dash-image.kiwi.ini]: All looks good to me, just this version is not aligned with the one in other images. Before, in pre-caasp v4 this was required to have the same major version as CaaSP, that's why all them are in the form of `4.0.x`. I believe this is not the case anymore and I think it only has effect on the resulting file name nowadays. However, just to be safe and in order to be coherent with other images, I'd say is better to set the version as `4.0.1`.

@jordi, this version was used by containment-rpm-docker to set the rpm package version, which is not used anymore in CaaSP, thus I believe we do not really care about that value, but I am not 100% sure it is not used in other places.

@sysrich, I think you are still using wrapped images in RPMs for Kubic, is the package version required to follow some versioning scheme?

Just as a not I noted heapster image is also having version 1.0.1, so far no one complained.

**Comment by thkukuk** [comment]
On Fri, May 25, David Cassany Viladomat wrote:

> @sysrich, I think you are still using wrapped images in RPMs for Kubic, is the
> package version required to follow some versioning scheme?

Kubic is using rolling upgrades, so version numbers should be increasing,
else you are free to choose what fits.
For Kubic/caasp4 we did already have some version downgrades, the pause
container has now version 0.1, e.g.

  Thorsten

-- 
Thorsten Kukuk, Distinguished Engineer, Senior Architect SLES & CaaSP
SUSE LINUX GmbH, Maxfeldstr. 5, 90409 Nuernberg, Germany
GF: Felix Imendoerffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nuernberg)

File: kube-dash-image/kube-dash-image.kiwi.ini (hunk 1 above)
Comment (jordimassaguerpla) [reply kube-dash-image/kube-dash-image.kiwi.ini]: Please use 4.0.* as version to be coherent.

**Comment by jordimassaguerpla** [comment]
pause container is an exception.

**Comment by nanoscopic** [comment]
I will update the version to 4.0.1; as noted it is this version number in order to match heapster, as this image does depend on heapster to function.

**Comment by nanoscopic** [comment]
I have updated the version to 4.0.1 for both the dashboard image and the heapster image.

**Comment by Jordi** [comment]
I'm receiving these emails, but I'm not the "Jordi" you are looking for.
Please, update email address. Thanks.

**Pull request merged**